Client/src/components/NavBar.jsx

```diff
@@ -23,41 +22,41 @@ const NavBar = () => {
     setActive(e.target.innerText);
   };
 
-  // const logOut = async () => {
-  //   let myHeaders = new Headers();
-  //   myHeaders.append("Content-Type", "application/json");
-
-  //   let requestOptions = {
-  //     method: "POST",
-  //     headers: myHeaders,
-  //     redirect: "follow",
-  //     credentials: "include",
-  //   };
-
-  //   let user = cookies.get("user");
-
-  //   if (user) {
-  //     fetch(
-  //       "https://technoob-staging.azurewebsites.net/api/v1/authenticate/logout",
-  //       requestOptions
-  //     )
-  //       .then((response) => {
-  //         if (response.status === 200) {
-  //           setIsLoggedIn(false);
-  //           setUserProfile(null);
-  //           cookies.remove("user");
-  //         }
-  //         return response.json();
-  //       })
-  //       .catch((error) => {
-  //         console.log("error", error);
-  //       });
-  //   }
-  // };
-  // const handleClick = async () => {
-  //   await logOut();
-  //   navigate("/Home");
-  // };
+  const logOut = async () => {
```

Comment:
Kindly rework this logout function to use the serverAPI file and toast just like we do with login.

Reply:
Ooh okay . I didn't even touch it . I met it that way and I didn't even bother to check it . But I will look into it